src/js/containers/SubMenu.js

```diff
@@ -2,8 +2,10 @@ import { connect } from 'react-redux'
 import VScrollMenu from '../VScrollMenu'
```

Comment:
Overall design comments:

* Seems this might be an existing issue, but there really is no indication of which menu items are submenus in a TILED layout
* As far as implementing driver distraction limitations, I think we should either grey out the options that go deeper than the limit or omit them from the list entirely, leaving only commands at the final allowed level. It would be nice if there were some indication that items were omitted or disabled for safety purposes.

Reply:
Made updates to this issue in 
06c0b3d